VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -101,65 +101,62 @@
 
             <LinearLayout
                 android:id="@+id/last_access_bar"
-                android:orientation="vertical"
                 style="@style/last_access_bar_style"
-                android:visibility="gone">
```

Comment:
Are you using the user **staff** ? cuz he's been to everywhere :wink: 
Otherwise, last time I checked after making a new user, Last access bar didnt show up in a course until I visited a `section`.